tests/library/browsercontext-pageerror-event.spec.ts

```diff
@@ -1,5 +1,6 @@
+
```

Comment:
Let's keep `page-event-pageerror.spec.ts` as is, and instead add a test to `browsercontext-events.spec.ts`.